### PR TITLE
[zh-cn]: update the translation of "Introduction to game development"

### PR DESCRIPTION
**Pull request by tianyeeT**

### CODE REVIEW

**Review by yin1999 — CHANGES_REQUESTED** [review]
麻烦修一下损坏的链接：https://pr21937.content.dev.mdn.mozit.cloud/zh-CN/docs/Games/Introduction#_flaws

File: files/zh-cn/games/introduction/index.md [hunk 1]
```diff
@@ -1,56 +1,60 @@
...
+  sourceCommit: e72890bafe775a38620def9a74beda8cf9c47411
 ---
 
 {{GamesSidebar}}
 
-现代的 web 已经高速发展成为一个可行可靠的平台，它不仅能够用来创建高质量的酷炫游戏，同时也能够用来发布和传播这些游戏。
+现代 web 已经高速发展成为一个可行可靠的平台，它不仅能够用来创建高质量的酷炫游戏，同时也能够用来发布和传播这些游戏。
 
-采用现代网页技术和较新的浏览器，完全有可能做出令人印象深刻的顶级页面游戏。它能够制作的游戏种类可以和桌面端以及原生系统相当。我们这里所说的，并不是很久之前就采用 Flash® 制作出的简单卡牌游戏或者多人社交游戏。而是牛逼的 3D 动作射击游戏，RPG 游戏等等。得益于 [JavaScript](/zh-CN/docs/JavaScript) 实时编译技术性能的大幅提升，以及新开放的 API。在制作运行在浏览器（或者是基于类似 [Firefox OS](/zh-CN/docs/Mozilla/Firefox_OS) 的 [HTML5](/zh-CN/docs/HTML/HTML5)技术支持的设备）上的游戏时，我们不用妥协。
+它能够制作的游戏种类可以和桌面端以及原生系统相当。采用现代网页技术和较新的浏览器，完全有可能做出令人印象深刻的顶级页面游戏。我们这里所说的，并不是很久之前就采用 Flash® 制作出的简单卡牌游戏或者多人社交游戏，而是 3D 动作射击游戏、RPG 游戏等等。得益于 [JavaScript](/zh-CN/docs/JavaScript) 实时编译技术性能的大幅提升，以及新开放的 API。在制作运行在浏览器（或者是基于 [HTML5](/zh-CN/docs/HTML/HTML5) 技术支持的设备）上的游戏时，我们不用妥协。
```
Comment (tianyeeT) [inline files/zh-cn/games/introduction/index.md:12]:
```suggestion
它能够制作的游戏种类可以和桌面端以及原生系统相当。采用现代网页技术和较新的浏览器，完全有可能做出令人印象深刻的顶级页面游戏。我们这里所说的，并不是很久之前就采用 Flash® 制作出的简单卡牌游戏或者多人社交游戏，而是 3D 动作射击游戏、RPG 游戏等等。得益于 [JavaScript](/zh-CN/docs/Web/JavaScript) 实时编译技术性能的大幅提升，以及新开放的 API。在制作运行在浏览器（或者是基于 [HTML5](/zh-CN/docs/Glossary/HTML5) 技术支持的设备）上的游戏时，我们不用妥协。
```

File: files/zh-cn/games/introduction/index.md [hunk 2]
```diff
@@ -1,56 +1,60 @@
...
 | **音频** | [Web Audio API](/zh-CN/docs/Web/API/Web_Audio_API)                                                                                                                                                                                                                        |
-| **图形** | [WebGL](/zh-CN/docs/WebGL) ([OpenGL ES](http://www.khronos.org/opengles/) 2.0)                                                                                                                                                                                            |
-| **输入** | [Touch events](/zh-CN/docs/DOM/Touch_events), [Gamepad API](/zh-CN/docs/Web/API/Gamepad_API/Using_the_Gamepad_API), 设备传感器，[WebRTC](/zh-CN/docs/WebRTC), [Full Screen API](/zh-CN/docs/Web/API/Fullscreen_API), [Pointer Lock API](/zh-CN/docs/API/Pointer_Lock_API) |
+| **图形** | [WebGL](/zh-CN/docs/WebGL) ([OpenGL ES](https://www.khronos.org/opengles/) 2.0)                                                                                                                                                                                           |
```
Comment (tianyeeT) [inline files/zh-cn/games/introduction/index.md:21]:
```suggestion
| **图形** | [WebGL](/zh-CN/docs/Web/API/WebGL_API) ([OpenGL ES](https://www.khronos.org/opengles/) 2.0)                                                                                                                                                                                           |
```

File: files/zh-cn/games/introduction/index.md [hunk 3]
```diff
@@ -1,56 +1,60 @@
...
+| **输入** | [Touch events](/zh-CN/docs/DOM/Touch_events)、[Gamepad API](/zh-CN/docs/Web/API/Gamepad_API/Using_the_Gamepad_API)、设备传感器、[WebRTC](/zh-CN/docs/WebRTC)、[Full Screen API](/zh-CN/docs/Web/API/Fullscreen_API)、[Pointer Lock API](/zh-CN/docs/API/Pointer_Lock_API) |
```
Comment (tianyeeT) [inline files/zh-cn/games/introduction/index.md:22]:
```suggestion
| **输入** | [Touch events](/zh-CN/docs/Web/API/Touch_events)、[Gamepad API](/zh-CN/docs/Web/API/Gamepad_API/Using_the_Gamepad_API)、设备传感器、[WebRTC](/zh-CN/docs/Web/API/WebRTC_API)、[Full Screen API](/zh-CN/docs/Web/API/Fullscreen_API)、[Pointer Lock API](/zh-CN/docs/Web/API/Pointer_Lock_API) |
```

File: files/zh-cn/games/introduction/index.md [hunk 4]
```diff
@@ -1,56 +1,60 @@
...
 | **语言** | [JavaScript](/zh-CN/docs/JavaScript) (或是 C/C++ 使用 [Emscripten](https://github.com/kripken/emscripten/wiki) 来编译成 JavaScript)                                                                                                                                       |
```
Comment (tianyeeT) [inline files/zh-cn/games/introduction/index.md:23]:
```suggestion
| **语言** | [JavaScript](/zh-CN/docs/Web/JavaScript) (或是 C/C++ 使用 [Emscripten](https://github.com/kripken/emscripten/wiki) 来编译成 JavaScript)                                                                                                                                       |
```

File: files/zh-cn/games/introduction/index.md [hunk 5]
```diff
@@ -1,56 +1,60 @@
...
-| **网络** | [WebRTC](/zh-CN/docs/WebRTC) 和/或 [WebSockets](/zh-CN/docs/Web/API/WebSockets_API)                                                                                                                                                                                       |
-| **存储** | [IndexedDB](/zh-CN/docs/IndexedDB) 或是 "云 (存储)"                                                                                                                                                                                                                       |
-| **Web**  | [HTML](/zh-CN/docs/HTML), [CSS](/zh-CN/docs/CSS), [SVG](/zh-CN/docs/SVG), [Social API](/zh-CN/docs/Mozilla/Projects/Social_API) (还有其他很多很多东西！)                                                                                                                  |
+| **网络** | [WebRTC](/zh-CN/docs/WebRTC) 或 [WebSockets](/zh-CN/docs/Web/API/WebSockets_API)                                                                                                                                                                                          |
```
Comment (tianyeeT) [inline files/zh-cn/games/introduction/index.md:24]:
```suggestion
| **网络** | [WebRTC](/zh-CN/docs/Web/API/WebRTC_API) 或 [WebSockets](/zh-CN/docs/Web/API/WebSockets_API)                                                                                                                                                                                          |
```

File: files/zh-cn/games/introduction/index.md [hunk 6]
```diff
@@ -1,56 +1,60 @@
...
+| **存储** | [IndexedDB](/zh-CN/docs/IndexedDB) 或“云（存储）”                                                                                                                                                                                                                         |
```
Comment (tianyeeT) [inline files/zh-cn/games/introduction/index.md:25]:
```suggestion
| **存储** | [IndexedDB](/zh-CN/docs/Web/API/IndexedDB_API) 或“云（存储）”                                                                                                                                                                                                                         |
```

File: files/zh-cn/games/introduction/index.md [hunk 7]
```diff
@@ -1,56 +1,60 @@
...
+| **Web**  | [HTML](/zh-CN/docs/HTML)、[CSS](/zh-CN/docs/CSS)、[SVG](/zh-CN/docs/SVG)、[Social API](/zh-CN/docs/Mozilla/Projects/Social_API) (还有其他很多东西！)                                                                                                                      |
```
Comment (tianyeeT) [inline files/zh-cn/games/introduction/index.md:26]:
```suggestion
| **Web**  | [HTML](/zh-CN/docs/Web/HTML)、[CSS](/zh-CN/docs/Web/CSS)、[SVG](/zh-CN/docs/Web/SVG)（还有其他很多东西！）                                                                                                                   |
```

File: files/zh-cn/games/introduction/index.md [hunk 8]
```diff
@@ -1,56 +1,60 @@
...
 
+- [Fetch API](/zh-CN/docs/Web/API/Fetch_API)
+  - : 从 web 服务器发送和接收任何类型的数据，比如下载新的游戏关卡和艺术资源，或者传输非实时的游戏状态信息。
 - [Full Screen API](/zh-CN/docs/Web/API/Fullscreen_API)
   - : 这个简单的 API 能够让你的游戏占据整个屏幕，从而使玩家沉浸在动作中
 - [Gamepad API](/zh-CN/docs/Web/API/Gamepad_API/Using_the_Gamepad_API)
   - : 如果你想你的用户能够使用游戏手柄或其他游戏控制器来控制游戏，你需要这个 API
-- [HTML](/zh-CN/docs/HTML) and [CSS](/zh-CN/docs/CSS)
-  - : 二者合璧，可以构建，设计并对你的游戏界面布局，HTML 有一个提供 2D 图形的元素，即{{HTMLElement("canvas")}}
+- [HTML](/zh-CN/docs/HTML) 和 [CSS](/zh-CN/docs/CSS)
```
Comment (tianyeeT) [inline files/zh-cn/games/introduction/index.md:50]:
```suggestion
- [HTML](/zh-CN/docs/Web/HTML) 和 [CSS](/zh-CN/docs/Web/CSS)
```

File: files/zh-cn/games/introduction/index.md [hunk 9]
```diff
@@ -1,56 +1,60 @@
...
+  - : 二者合璧，可以构建、设计并对你的游戏界面进行布局，HTML 有一个提供 2D 图形的元素，即 {{HTMLElement("canvas")}}。
 - [HTML audio](/zh-CN/docs/HTML/Element/audio)
```
Comment (tianyeeT) [inline files/zh-cn/games/introduction/index.md:52]:
```suggestion
- [HTML audio](/zh-CN/docs/Web/HTML/Element/audio)
```

File: files/zh-cn/games/introduction/index.md [hunk 10]
```diff
@@ -1,56 +1,60 @@
...
-  - : {{HTMLElement("audio")}} 元素可以用来播放一些简单的音效和音乐。如果你想要更多的参与，可以学习[Web Audio API](/zh-CN/docs/Web/API/Web_Audio_API) 来深入了解音频处理的力量！
+  - : {{HTMLElement("audio")}} 元素可以用来播放一些简单的音效和音乐。如果你想要更多的参与，可以学习 [Web Audio API](/zh-CN/docs/Web/API/Web_Audio_API) 来深入了解音频处理的力量！
 - [IndexedDB](/zh-CN/docs/IndexedDB)
```
Comment (tianyeeT) [inline files/zh-cn/games/introduction/index.md:54]:
```suggestion
- [IndexedDB](/zh-CN/docs/Web/API/IndexedDB_API)
```

File: files/zh-cn/games/introduction/index.md [hunk 11]
```diff
@@ -1,56 +1,60 @@
...
   - : 一个强大的数据存储 API，用来在电脑或者设备上保存用户自己的数据。一个很好的方法用来保存游戏的状态和其他最近的信息，这样在需要的时候不用每次重新下载。也可以用来让你的游戏即使用户没有链接到网络也能继续玩（例如在飞机上的数小时）。
 - [JavaScript](/zh-CN/docs/JavaScript)
```
Comment (tianyeeT) [inline files/zh-cn/games/introduction/index.md:56]:
```suggestion
- [JavaScript](/zh-CN/docs/Web/JavaScript)
```

File: files/zh-cn/games/introduction/index.md [hunk 12]
```diff
@@ -1,56 +1,60 @@
...
-  - : JavaScript 是网络上使用的编程语言，在现代浏览器中正在快速发展，而且一直在快速发展。使用它的力量为你的游戏编写代码，或者使用[Emscripten](https://github.com/kripken/emscripten/wiki)或[Asm.js](http://asmjs.org/spec/latest/)等技术轻松移植你现有的游戏。
+  - : JavaScript 是 web 所使用的编程语言，在现代浏览器中正在快速发展，而且一直在快速发展。使用它的力量为你的游戏编写代码，或者使用 [Emscripten](https://github.com/kripken/emscripten/wiki) 或 [Asm.js](http://asmjs.org/spec/latest/) 等技术轻松移植你现有的游戏。
 - [Pointer Lock API](/zh-CN/docs/API/Pointer_Lock_API)
```
Comment (tianyeeT) [inline files/zh-cn/games/introduction/index.md:58]:
```suggestion
- [Pointer Lock API](/zh-CN/docs/Web/API/Pointer_Lock_API)
```

File: files/zh-cn/games/introduction/index.md [hunk 13]
```diff
@@ -1,56 +1,60 @@
...
   - : 指针锁定 API 允许你在游戏界面中锁定鼠标或其他指针设备，以便你不用绝对定位光标就可以获得坐标变化值，从而准确地判断用户正在做什么，并且还可以防止用户意外地进入另一块屏幕或别的什么地方，从而导致误操作。
 - [SVG](/zh-CN/docs/SVG) (可缩放矢量图形)
```
Comment (tianyeeT) [inline files/zh-cn/games/introduction/index.md:60]:
```suggestion
- [SVG](/zh-CN/docs/Web/SVG) (可缩放矢量图形)
```

File: files/zh-cn/games/introduction/index.md [hunk 14]
```diff
@@ -60,12 +64,10 @@ slug: Games/Introduction
 - [Web Audio API](/zh-CN/docs/Web/API/Web_Audio_API)
   - : 这个 API 用于控制 JavaScript 代码中的音频的回放，合成和处理，使你可以创建出色的音效，以及实时播放和操作音乐。
 - [WebGL](/zh-CN/docs/WebGL)
```
Comment (tianyeeT) [inline files/zh-cn/games/introduction/index.md:66]:
```suggestion
- [WebGL](/zh-CN/docs/Web/API/WebGL_API)
```

File: files/zh-cn/games/introduction/index.md [hunk 15]
```diff
@@ -60,12 +64,10 @@ slug: Games/Introduction
...
-  - : 允许你从 Web 内容创建高性能，硬件加速的 3D（和 2D）图形。这是一个 Web 支持的[OpenGL ES](https://www.khronos.org/opengles/) 2.0 实现。
+  - : 允许你从 Web 内容创建高性能，硬件加速的 3D（和 2D）图形。这是一个 web 支持的 [OpenGL ES](https://www.khronos.org/opengles/) 2.0 实现。
 - [WebRTC](/zh-CN/docs/WebRTC)
```
Comment (tianyeeT) [inline files/zh-cn/games/introduction/index.md:68]:
```suggestion
- [WebRTC](/zh-CN/docs/Web/API/WebRTC_API)
```

File: files/zh-cn/games/introduction/index.md [hunk 16]
```diff
@@ -60,12 +64,10 @@ slug: Games/Introduction
...
-  - : WebRTC（实时通信）API 使你能够控制音频和视频数据，包括远程会议以及两个用户之间来回传输其他应用程序数据。希望你的玩家能够在殴打怪物的同时互相交流？这是你的 API，快使用它吧。
+  - : WebRTC（实时通信）API 使你能够控制音频和视频数据，包括远程会议以及两个用户之间来回传输其他应用程序数据。希望你的玩家能够在殴打怪物的同时互相交流？这正是为你所准备的 API，快使用它吧。
 - [WebSockets](/zh-CN/docs/Web/API/WebSockets_API)
```
Comment (tianyeeT) [inline files/zh-cn/games/introduction/index.md:70]:
```suggestion
- [WebSockets](/zh-CN/docs/Web/API/Web_Workers_API/Using_web_workers)
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**